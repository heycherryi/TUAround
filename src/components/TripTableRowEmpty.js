import React from 'react';

function TripTableRowEmpty() {
  return (
    <tr className="bg-info text-bg-info">
      <td colSpan="99" className="text-center p-2">No scheduled trips.</td>
    </tr>
  );
}

export default TripTableRowEmpty;
