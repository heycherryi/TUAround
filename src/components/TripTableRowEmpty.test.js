/* globals test, expect */

import React from 'react';
import { render, screen } from '@testing-library/react';
import TripTableRowEmpty from './TripTableRowEmpty';

test('renders TripTableRowEmpty', () => {
  const { container } = render(
    <table>
      <tbody>
        <TripTableRowEmpty />
      </tbody>
    </table>,
  );
  expect(screen.getByText('No scheduled trips.')).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
