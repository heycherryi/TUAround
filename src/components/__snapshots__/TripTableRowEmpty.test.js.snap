// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TripTableRowEmpty 1`] = `
<div>
  <table>
    <tbody>
      <tr
        class="bg-info text-bg-info"
      >
        <td
          class="text-center p-2"
          colspan="99"
        >
          No scheduled trips.
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
